hackathon_site/event/test_api.py

```diff
@@ -370,6 +370,150 @@ def test_name_search_filter(self):
         self.assertCountEqual(returned_ids, [self.team2.team_code])
 
 
+class ProfileDetailViewTestCase(SetupUserMixin, APITestCase):
```

Comment:
Could you also add a quick round of tests for each of these views that modifying the read only fields throws a validation error (status 400)? You can just do a single test method for each with a for loop or something to make the requests

Reply:
Per the comments above, it looks like DRF's default behaviour is to just ignore the read only fields.

That's fine by me - so we can either remove those tests (sorry :( ) and add the one above that would assert on `CurrentTeamOrderListView.read_only_fields` directly, or you could modify the tests you added to make sure that the read only fields weren't modified (but the former is probably easier).